roxie/ccd/ccdserver.cpp

```diff
@@ -7238,6 +7238,7 @@ class CRoxieServerDedupKeepRightActivity : public CRoxieServerDedupActivity
 
     virtual void start(unsigned parentExtractSize, const byte *parentExtract, bool paused)
     {
+        assertex(numToKeep <= 1);
```

Comment:
If an error can be provoked by SENSIBLE user CODE then it should be caught and throw with a better error message.